src/core/types/General.ts

```diff
@@ -8,4 +8,6 @@ export type DataType = 'amount' | 'percent' | 'usd';
 
 export type Route = 'portfolio' | 'vaults' | 'vault' | 'labs';
 
+export type ServiceId = 'zapper' | 'tenderly' | 'notify';
```

Comment:
I think we should be a bit more explicit on what type of services these are
```suggestion
export type ExternalServiceId = 'zapper' | 'tenderly' | 'notify';
```

Reply:
Agree, changed to better naming here https://github.com/yearn/yearn-finance-v3/pull/674/commits/61331273545f2bb6eb771a5df386c3825a40c640